### DISCUSSION TITLE
Adding Vale to your workflow

### DISCUSSION THREAD

**Discussion by Gerry-Forde** [redhat-documentation/vale-at-red-hat#41] [Ideas]
It would be useful to provide opinionated workflows which demonstrate different ways that Vale may be added to a writers tool chain. 

Some may just consider using Vale as an IDE plugin to improve their technical writing skills, others may see it a component of a full CI workflow for their teams or as a validation tool for specific documentation.

Either way there are lots of possibilities and documentation of various implementation strategies may encourage more general adoption of Vale.

**Comment by themr0c** [comment]
A good start:

* https://vale-at-red-hat.github.io/vale-at-red-hat/docs/end-user-guide/using-vale-cli/
* https://vale-at-red-hat.github.io/vale-at-red-hat/docs/end-user-guide/using-vale-in-the-ide/
* https://vale-at-red-hat.github.io/vale-at-red-hat/docs/end-user-guide/reporting-vale-alerts/
* https://vale-at-red-hat.github.io/vale-at-red-hat/docs/end-user-guide/defining-a-vale-onboarding-strategy/
* https://vale-at-red-hat.github.io/vale-at-red-hat/docs/end-user-guide/adding-vale-configuration-to-a-project/
* https://vale-at-red-hat.github.io/vale-at-red-hat/docs/end-user-guide/creating-a-vocabulary-for-a-project/
* https://vale-at-red-hat.github.io/vale-at-red-hat/docs/end-user-guide/using-vale-github-action/
* https://vale-at-red-hat.github.io/vale-at-red-hat/docs/end-user-guide/using-vale-with-a-continuous-integration-service/
